src/Compilers/CSharp/Portable/Parser/LanguageParser.cs

```diff
@@ -4470,6 +4510,7 @@ private void ParseVariableDeclarators(TypeSyntax type, VariableFlags flags, Sepa
             // the reported errors should take into consideration whether or not one expects them in the current context.
             bool variableDeclarationsExpected =
                 parentKind != SyntaxKind.NamespaceDeclaration &&
+                parentKind != SyntaxKind.SingleLineNamespaceDeclaration &&
```

Comment:
Is there a test for this change?

Reply:
@RikkiGibson to ensure we've addressed this

Reply:
Adding the tests.